lib/mix/tasks/appsignal.demo.ex

```diff
@@ -8,5 +9,7 @@ defmodule Mix.Tasks.Appsignal.Demo do
     Appsignal.Demo.create_transaction_performance_request
     Appsignal.Demo.create_transaction_error_request
     Appsignal.stop(nil)
+    Logger.info("Demonstration sample data sent!")
+    Logger.info("It may take about a minute for the data to appear on AppSignal.com/accounts")
```

Comment:
:+1: If you put in the full link `https://appsignal.com/accounts`, it will be clickable from most terminals :)